test/datasets/test_data_frame_text_benchmark.py

```diff
@@ -0,0 +1,105 @@
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name': 'product_sentiment_machine_hack'
+            })
+            assert datasets[1] == ('MultimodalTextBenchmark', {
+                'name': 'news_channel'
+            })
+        elif scale == 'medium':
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name': 'wine_reviews'
+            })
+            assert datasets[1] == ('HuggingFaceDatasetDict', {
+                'path':
+                'maharshipandya/spotify-tracks-dataset',
+                'columns': [
+                    'artists', 'album_name', 'track_name', 'popularity',
+                    'duration_ms', 'explicit', 'danceability', 'energy', 'key',
+                    'loudness', 'mode', 'speechiness', 'acousticness',
+                    'instrumentalness', 'liveness', 'valence', 'tempo',
+                    'time_signature', 'track_genre'
+                ],
+                'target_col':
+                'track_genre',
+            })
+        elif scale == 'large':
+            assert datasets[0] == ('AmazonFineFoodReviews', {})
+    elif task_type == TaskType.REGRESSION:
+        if scale == 'small':
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name':
+                'google_qa_answer_type_reason_explanation'
+            })
+            assert datasets[1] == ('MultimodalTextBenchmark', {
+                'name':
+                'google_qa_question_type_reason_explanation'
+            })
+            assert datasets[2] == ('MultimodalTextBenchmark', {
+                'name': 'bookprice_prediction'
+            })
+            assert datasets[3] == ('MultimodalTextBenchmark', {
+                'name': 'jc_penney_products'
+            })
+            assert datasets[4] == ('MultimodalTextBenchmark', {
+                'name': 'women_clothing_review'
+            })
+            assert datasets[5] == ('MultimodalTextBenchmark', {
+                'name': 'news_popularity2'
+            })
+        elif scale == 'medium':
+            assert len(datasets) == 0
+        elif scale == 'large':
+            assert datasets[0] == ('Mercari', {})
+
+
+def test_data_frame_text_benchmark_object():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        dataset = DataFrameTextBenchmark(
+            root=temp_dir,
+            task_type=TaskType.MULTICLASS_CLASSIFICATION,
+            scale='small',
+            idx=1,
+            col_to_text_embedder_cfg=TextEmbedderConfig(
+                text_embedder=HashTextEmbedder(10)),
+        )
+    assert str(dataset) == ("DataFrameTextBenchmark(\n"
+                            "  task_type=multiclass_classification,\n"
+                            "  scale=small,\n"
+                            "  idx=1,\n"
+                            "  cls=MultimodalTextBenchmark()\n"
+                            ")")
+    assert dataset.num_rows == 25355
+    dataset.materialize()
```

Comment:
super nit: since this one involves downloading, do you mind choosing the smallest dataset? I saw there is one with `6,364` rows.

Reply:
ok